### PR TITLE
feat: implement pom to framework

### PR DESCRIPTION
**Pull request by iEnergyy**
**implemented POM to the framework.**
- added two examples for test structure.
- added two page objects pages.

### CODE REVIEW

**Review by moisesisaias — DISMISSED** [review]
Everything looks good, I would just add the page objects and test under a sub-folder with a name related to the sauce demo web site, this way we could test multiple sites using this repo.

File: cypress/e2e/UI/Tests/LoginTests.cy.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+/// <reference types="cypress" />
+
+import InventoryPage from '../PageObjects/InventoryPage';
+import LoginPage from '../PageObjects/LoginPage';
+
+describe('Login test', () => {
+
+    it('Logs in successfully to the web app', () => {
+        LoginPage.Login('standard_user', 'secret_sauce');
```
Comment (moisesisaias) [inline cypress/e2e/UI/Tests/LoginTests.cy.js]: I think we should put these login values in a shared constant or json file.

**Comment by iEnergyy** [comment]
> Everything looks good, I would just add the page objects and test under a sub-folder with a name related to the sauce demo web site, this way we could test multiple sites using this repo.

made a commit to create the folder and move tests and page objects

**Review by moisesisaias — APPROVED** [review]

File: cypress/e2e/UI/Tests/LoginTests.cy.js (hunk 1 above)
Comment (moisesisaias) [reply cypress/e2e/UI/Tests/LoginTests.cy.js]: @iEnergyy, vamos a hacer esto en otro PR.

**Pull request merged**